core/src/test/scala/akka/persistence/r2dbc/journal/PersistTimestampSpec.scala

```diff
@@ -80,7 +80,11 @@ class PersistTimestampSpec
               Row(
                 pid = row.get("persistence_id", classOf[String]),
                 seqNr = row.get[java.lang.Long]("seq_nr", classOf[java.lang.Long]),
-                dbTimestamp = row.get("db_timestamp", classOf[Instant]),
+                dbTimestamp = if (settings.dialectName == "sqlserver") {
+                  row.get("db_timestamp", classOf[LocalDateTime]).atZone(ZoneId.systemDefault()).toInstant
```

Comment:
I haven't read up on sqlserver timestamps, but the principle should be that we always use UTC

Reply:
Yes this is what I have done. I just missed migrating it in this test, good catch!